templates/Datashades-OpsWorks-CKAN-Instances.cfn.yml.j2

```diff
@@ -18,21 +18,31 @@ Parameters:
       - STAGING
       - PROD
   ServicesEC2Size:
-    Description: Select EC2 Instance Size.
+    Description: Select EC2 Instance Size for Solr
     Type: String
     Default: t3.small
     AllowedValues:
       - t3.micro
       - t3.small
       - t3.medium
       - t3.large
+      - t3a.micro
```

Comment:
Is the t3a family covered by our existing instance reservations?

Reply:
nope, is what started me fixing the script as it was rejected. manually altering the instance from ec2 console may have been done.

Reply:
sorry, mis-read the question. As this time, we do not have RI's purchased, we moved to 'savings plans' which does not lock us into any instance type. We are free to move to any option.  The only RI's we do have is RDS until may this year.